package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@msgpack/msgpack",
-  "version": "2.8.0",
+  "name": "msgpack",
+  "version": "1.0.0",
   "description": "MessagePack for ECMA-262/JavaScript/TypeScript",
   "author": "The MessagePack community",
```

Comment:
Let's also update the other keys in `package.json`, e.g. `description` and `repository`. Let's make sure to mention that it's a fork in the description.